eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -0,0 +1,145 @@
+<#
+.SYNOPSIS
+Updates package README.md for publishing to docs.microsoft.com
+
+.DESCRIPTION
+Given a PackageInfo .json file, format the package README.md file with metadata
+and other information needed to release reference docs: 
+
+* Adjust README.md content to include metadata
+* Insert the package verison number in the README.md title 
+* Copy file to the appropriate location in the documentation repository
+* Copy PackageInfo .json file to the metadata location in the reference docs
+  repository. This enables the Docs CI build to onboard packages which have not
+  shipped and for which there are no entries in the metadata CSV files.
+
+.PARAMETER ArtifactLocation
```

Comment:
This name doesn't seem to make since in this context. We should rename it to PackageInfo or something closer to what it represents.